src/Hl7.Fhir.InterfaceApplier.CLI/Implementations/InterfaceApplierManager.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Hl7.Fhir.InterfaceApplier.CLI.Abstractions;
+using Microsoft.Extensions.Logging;
+using System.Reflection;
+using System.Runtime.Loader;
+
+namespace Hl7.Fhir.InterfaceApplier.CLI.Implementations;
+
+public class InterfaceApplierManager : IInterfaceApplierManager
+{
+    #region Private Fields
+
+    private readonly IInterfaceDiscoveryService _interfaceDiscoveryService;
+    private readonly IProjectManager _projectManager;
+    private readonly IClassDiscoveryService _classDiscoveryService;
+    private readonly IInterfaceApplierService _interfaceApplierService;
+    private readonly ILogger<InterfaceApplierManager> _logger;
+
+    #endregion
+
+    #region Constructors
+
+    public InterfaceApplierManager(
+        IProjectManager projectManager,
+        IInterfaceDiscoveryService interfaceDiscoveryService,
+        IClassDiscoveryService classDiscoveryService,
+        IInterfaceApplierService interfaceApplierService,
+        ILogger<InterfaceApplierManager> logger)
+    {
+        _projectManager = projectManager;
+        _interfaceDiscoveryService = interfaceDiscoveryService;
+        _classDiscoveryService = classDiscoveryService;
+        _interfaceApplierService = interfaceApplierService;
+        _logger = logger;
+    }
+
+    #endregion
+
+    #region Public Methods
+
+    public void Run()
+    {
+        var fhirBaseProjectDirectory = _projectManager.GetFhirBaseProjectDirectory();
+        var fhirConformanceProjectDirectory = _projectManager.GetFhirConformanceProjectDirectory();
+
+        foreach (var projectVersionPath in _projectManager.GetFhirVersionProjectFilePaths())
+        {
+            applyInterfacesToProject(projectVersionPath, fhirBaseProjectDirectory, fhirConformanceProjectDirectory);
+        }
+    }
+
+    #endregion
+
+    #region Private Methods
+
+    private void applyInterfacesToProject(string projectFilePath, string fhirBaseProjectDirectory, string fhirConformanceProjectDirectory)
```

Comment:
Resharper stubbornly refused to make private methods CamelCase. I'm used to have methods for all visibility starting with an upper case. Is this a configuration in the project/solution? I've never seen that.